apis/core.oam.dev/v1alpha1/garbagecollect_types.go

```diff
@@ -47,6 +47,7 @@ type GarbageCollectPolicyRule struct {
 // GarbageCollectPolicyRuleSelector select the targets of the rule
 type GarbageCollectPolicyRuleSelector struct {
 	TraitTypes []string `json:"traitTypes"`
+	CompTypes  []string `json:"compTypes"`
```

Comment:
Add some comments here. If compTypes and traitTypes are both used, the combination logic is `or`.

Reply:
done